### PR TITLE
Coveralls coverage

### PR DESCRIPTION
**Pull request by anais-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/coveralls-coverage/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/coveralls-coverage/docs/

File: _data/nav.yml [hunk 1]
```diff
@@ -232,6 +232,8 @@
       sub-pages:
         - title: Codecov Coverage Reports
           url: "/codecov-testing"
+        - title: Coveralls Coverage Reports
+          url: "/coveralls-testing"
```
Comment (kostis-codefresh) [inline _data/nav.yml:236]: You should also add it here https://codefresh.io/docs/docs/yaml-examples/examples/

File: _docs/yaml-examples/examples/coveralls-testing.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
+---
+title: "Coveralls Coverage Reports"
+description: "How to forward coverage reports to Coveralls"
+group: yaml-examples
+toc: true
+---
+
+[Coveralls](https://coveralls.io/) is a web service that allows users to track the code coverage of their application over time to optimise test coverage. This section details how coverage reports can be generated and forwarded to Coveralls with every Codefresh build.
```
Comment (kostis-codefresh) [inline _docs/yaml-examples/examples/coveralls-testing.md:8]: This is circular. I suggest "..is a web service that allows users to track the code coverage of their application over time in order to optimize the effectiveness of their unit tests".

File: _docs/yaml-examples/examples/coveralls-testing.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+{% endraw %}
+{% endhighlight %}
+
+Once you run the pipeline the steps will create the coverage report and forward it to Coveralls.
+
+{% include image.html 
+lightbox="true" 
+file="/images/testing/coveralls/coveralls-pipeline.png" 
+url="/images/testing/coveralls/coveralls-pipeline.png" 
+alt="Pipeline with Coveralls step"
```
Comment (kostis-codefresh) [inline _docs/yaml-examples/examples/coveralls-testing.md:175]: All the pictures of coveralls are really small. I suggest you use max-width 70% or 80% to make them bigger.

File: _docs/yaml-examples/examples/coveralls-testing.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+* A [Coveralls account](https://coveralls.io/) (free or enterprise) -- Note that all open-source projects are free on Coveralls
+* A testing tool added to your project that produces coverage reports
+
+Coveralls supports [22 different language integrations](https://docs.coveralls.io/about-coveralls). Each example provided in the official documentation suggests several coverage report tools that can be used in combination with Coveralls.
+
+You could try it out by cloning our [node example application](https://github.com/codefresh-contrib/coveralls-sample-app) that utilises [jest](https://jestjs.io/).
+
+## Prepare your Repository
+
+In case that you are using your own application, you have to make a few modifications to the repository. Please have a look at the Coveralls example section for other languages.
```
Comment (kostis-codefresh) [inline _docs/yaml-examples/examples/coveralls-testing.md:31]: I suggest "If you are using your own application as an example, you have to make ..."

File: _docs/yaml-examples/examples/coveralls-testing.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+First, install Coveralls in your project:
+{% highlight yaml %}
+{% raw %}
+npm install coveralls --save-dev
+{% endraw %}
+{% endhighlight %}
+
+Coveralls requires a [script](https://github.com/nickmerwin/node-coveralls) that takes standard input and sends it to coveralls.io to report your code coverage. Any coverage reports can be forwarded that are within a lcov data format (including mocha's LCOV reporter). For this, we are going to set-up a “bin” folder, and within the folder a coveralls.js file that contains the following content:
```
Comment (kostis-codefresh) [inline _docs/yaml-examples/examples/coveralls-testing.md:40]: What is the LCOV reporter? Can we add a link here?

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/coveralls-coverage/docs/

**Pull request merged**